src/content/docs/alerts/get-notified/notification-integrations.mdx

```diff
@@ -782,7 +783,9 @@ You can update ServiceNow attributes directly in the target table, such as **Inc
       alt="A screenshot completed authentication."
       src={accountsSlackAuthenticationCompleted}
     />
-  
+  <Callout variant="important">
+    Each Slack workspace has a unique destination per New Relic account.
```

Comment:
Didn't want this to be negative but the actual meaning is "We don't allow more than one destination per slack workspace per nr account" 
If the wording seems off, I'd appreciate some help changing it!